### ISSUE TITLE
check() fails, although Rcmd.exe check does not

### ISSUE THREAD

**Issue by strengejacke** [r-lib/devtools#1919]
When I use RStudio to check packages, with the option _Use devtools functions if package available_ checked, package checking fails and exits with error. However, when I check with `Rcmd.exe check`, everything is fine. The package I submitted to CRAN had no problems.

Could this be because of a warning that qpdf is not available?
``` 
WARNING
'qpdf' is needed for checks on size reduction of PDFs
```

I have this (and other issues with devtools) since the last update.

**Comment by russHyde** [comment]
Just had a similar thing on ubuntu where `R CMD build ...; R CMD check ` passed on a package that had failed with the `check` workflow in `rstudio`.  

The rstudio `check` was indicating that the parameters for a function had not been defined in my roxygen2 annotations; this was happening even after I added `@param` annotations for that function, after I renamed it (when the name of the initial function still threw an error in `check`) and after I `@noRd`ed the function.  

I believe that there was some stuff stored in my `.Rproj.user` that was causing the problem, because after renaming the function - and with the previously named function still throwing the error, but being absent from my `R/` directory - the only reference to the previous name of the function was in subdirectories of `.Rproj.user`.

After running R CMD build/check, the errors in rstudio's check disappeared. (devtools 2.0.1 / roxygen2 6.1.1 / rstudio 1.1.456)
